### DISCUSSION TITLE
Help Help

### DISCUSSION THREAD

**Discussion by Atw-Lee** [Tencent/wujie#614] [Q&A]
集成的应用中，含有[spreadJS的导出功能](https://demo.grapecity.com.cn/spreadjs/SpreadJSTutorial/features/workbook/excel-import-export/purejs)

如果是umi创建的项目中，在wujie中可以正常导出。
但是在cra创建的项目中，无法导出
猜测是webpack的配置问题，已经debugger三天了，寻求帮助！！！！

**Comment by fz6m** [comment]
cra 已经很久很久不维护了，dan 也表示后续会把 cra 淘汰掉，改成框架选择工具，如果自己解决不了 cra 的问题，使用 Umi 即可。
